.pre-commit-config.yaml

```diff
@@ -6,3 +6,13 @@ repos:
     hooks:
     - id: flake8
       additional_dependencies: [flake8-docstrings, flake8-debugger, flake8-bugbear]
+      args: [--max-complexity, "10"]
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        language_version: python3
+ci:
+  # To trigger manually, comment on a pull request with "pre-commit.ci autofix"
+  autofix_prs: false
+  skip: [bandit]
```

Comment:
This skip isn't needed since you aren't using `bandit`. I think pre-commit.ci will complain about this.